scripts/check_backend_associated_test_file.py

```diff
@@ -54,6 +54,7 @@
 
     # The files below are to be tested.
     # GSoC'22: Improve line and branch coverage for the backend and frontend.
+    'scripts/check_overall_backend_test_coverage.py',
```

Comment:
Can you also write tests for this file so that we know it's working correctly? Since we'll have to test it anyway, we might as well test it now, right?

Reply:
Done!